lib/beepboop-botkit.js

```diff
@@ -56,6 +56,11 @@ BeepBoopBotkit.prototype = {
         }
       })
 
+    this.controller
+      .on('rtm_close', function (worker) {
```

Comment:
do we need to trigger anything else on our end to handle to fact that this connection closed, i.e. retry logic, or is the plan to wait until that's added to botkit?